### PR TITLE
Azure deployments

### LINKED ISSUE #510: deploy: CockroachDB tutorial for Microsoft Azure
Based on `manual-deployment.md`.

### PR DESCRIPTION
**Pull request by sploiselle**
Covers both secure and insecure deployments, as well as adding necessary links.

Closes #510

---
This change is [Reviewable](https://reviewable.io/reviews/cockroachdb/docs/893)

### CODE REVIEW

File: deploy-cockroachdb-on-microsoft-azure-insecure.md [hunk 1]
```diff
@@ -0,0 +1,227 @@
...
+
+	~~~ shell
+	$ ssh <username>@<node1 external IP address>
+	~~~
+
+2.	Install the latest CockroachDB binary:
+	
+	~~~ shell
+	# Get the latest CockroachDB tarball.
+	$ wget https://binaries.cockroachdb.com/cockroach-{{site.data.strings.version}}.linux-amd64.tgz
```
Comment (mberhault) [inline deploy-cockroachdb-on-microsoft-azure-insecure.md]: shouldn't we be using `latest`?

File: deploy-cockroachdb-on-microsoft-azure-insecure.md [hunk 2]
```diff
@@ -0,0 +1,227 @@
...
+
+	# Extract the binary.
+	$ tar -xf cockroach-{{site.data.strings.version}}.linux-amd64.tgz  \
```
Comment (mberhault) [inline deploy-cockroachdb-on-microsoft-azure-insecure.md]: ditto

File: deploy-cockroachdb-on-microsoft-azure-insecure.md [hunk 3]
```diff
@@ -0,0 +1,227 @@
...
+
+	~~~
+	$ ssh <username>@<additional node external IP address>
+	~~~
+
+2.	Install CockroachDB from our latest binary:
+	
+	~~~ shell
+	# Get the latest CockroachDB tarball.
+	$ wget https://binaries.cockroachdb.com/cockroach-{{site.data.strings.version}}.linux-amd64.tgz
```
Comment (mberhault) [inline deploy-cockroachdb-on-microsoft-azure-insecure.md]: and here as well

File: deploy-cockroachdb-on-microsoft-azure.md [hunk 4]
```diff
@@ -0,0 +1,310 @@
...
+   ~~~ shell
+   $ cockroach cert create-client \
+   root \
+   --ca-cert=certs/ca.cert \
+   --ca-key=certs/ca.key \
+   --cert=certs/root.cert \
+   --key=certs/root.key
+   ~~~
+
+3. For each node, a create a node key pair issued for all common names you might use to refer to the node, including:
```
Comment (jseldess) [inline deploy-cockroachdb-on-microsoft-azure.md]: "a create a node" > "create a node"

**Comment by jseldess** [comment]
One more thought, though: Once the user/password PR is merged, you'll also have to update #2 under "Use the Database" to be:

[Create users](create-and-manage-users.html) and [grant them privileges](grant.html).

Might want to add that change to your other PR.

**Comment by sploiselle** [comment]
Review status: 0 of 9 files reviewed at latest revision, 6 unresolved discussions.

---

*[deploy-cockroachdb-on-microsoft-azure-insecure.md, line 8 at r1](https://reviewable.io:443/reviews/cockroachdb/docs/893#-KXH0-vuxpFb-lELqA8L-r1-8:-KXlCZ7bmzavsTyfS0vy:b-896fix) ([raw file](https://github.com/cockroachdb/docs/blob/c2719c99db4f24b5b2bc277585df5d810ebef19b/deploy-cockroachdb-on-microsoft-azure-insecure.md#L8)):*
<details><summary><i>Previously, mberhault (marc) wrote…</i></summary><blockquote>

s/oon/on/
</blockquote></details>

Done.

---

*[deploy-cockroachdb-on-microsoft-azure-insecure.md, line 109 at r1](https://reviewable.io:443/reviews/cockroachdb/docs/893#-KXH0-vuxpFb-lELqA8L-r1-109:-KXlCZqy68xGNrqOxu0-:b-896fix) ([raw file](https://github.com/cockroachdb/docs/blob/c2719c99db4f24b5b2bc277585df5d810ebef19b/deploy-cockroachdb-on-microsoft-azure-insecure.md#L109)):*
<details><summary><i>Previously, mberhault (marc) wrote…</i></summary><blockquote>

shouldn't we be using `latest`?
</blockquote></details>

Done.

---

*[deploy-cockroachdb-on-microsoft-azure-insecure.md, line 112 at r1](https://reviewable.io:443/reviews/cockroachdb/docs/893#-KXH0-vuxpFb-lELqA8L-r1-112:-KXlCaOA6I7fd0kEfmvE:b-896fix) ([raw file](https://github.com/cockroachdb/docs/blob/c2719c99db4f24b5b2bc277585df5d810ebef19b/deploy-cockroachdb-on-microsoft-azure-insecure.md#L112)):*
<details><summary><i>Previously, mberhault (marc) wrote…</i></summary><blockquote>

ditto
</blockquote></details>

Done.

---

*[deploy-cockroachdb-on-microsoft-azure-insecure.md, line 139 at r1](https://reviewable.io:443/reviews/cockroachdb/docs/893#-KXH0-vuxpFb-lELqA8L-r1-139:-KXlE0dioHxXNWkDO3JO:b-896fix) ([raw file](https://github.com/cockroachdb/docs/blob/c2719c99db4f24b5b2bc277585df5d810ebef19b/deploy-cockroachdb-on-microsoft-azure-insecure.md#L139)):*
<details><summary><i>Previously, mberhault (marc) wrote…</i></summary><blockquote>

and here as well
</blockquote></details>

Done.

---

*[deploy-cockroachdb-on-microsoft-azure-insecure.md, line 122 at r2](https://reviewable.io:443/reviews/cockroachdb/docs/893#-KXH0-vuxpFb-lELqA8L-r2-122:-KXlE-dBHh-Y1UQQFFTD:b-896fix) ([raw file](https://github.com/cockroachdb/docs/blob/76c2145fb5cfcb4fbe6efeab6c97a5be0498bec9/deploy-cockroachdb-on-microsoft-azure-insecure.md#L122)):*
<details><summary><i>Previously, jseldess wrote…</i></summary><blockquote>

We might want to note that you need to look at the virtual network to get the internal IP address. 
</blockquote></details>

Done.

---

*[deploy-cockroachdb-on-microsoft-azure.md, line 134 at r2](https://reviewable.io:443/reviews/cockroachdb/docs/893#-KXH0-vuxpFb-lELqA8M-r2-134:-KXlE5nSTs1AKd293KUp:b-896fix) ([raw file](https://github.com/cockroachdb/docs/blob/76c2145fb5cfcb4fbe6efeab6c97a5be0498bec9/deploy-cockroachdb-on-microsoft-azure.md#L134)):*
<details><summary><i>Previously, jseldess wrote…</i></summary><blockquote>

"a create a node" > "create a node"
</blockquote></details>

Done.

---


*Comments from [Reviewable](https://reviewable.io:443/reviews/cockroachdb/docs/893)*

**Comment by sploiselle** [comment]
Cannot repro HTML Proofer error on page. Errors are also in links currently on site, not newly introduced in this PR. Doing immediate follow-on investigation.

**Pull request merged**